packages/web/nextjs/index.ts

```diff
@@ -131,3 +136,73 @@ export function getStaticPathsFactory<C>(
     }
   };
 }
+
+export function createActionHandler<C>(fetcher: Fetcher<C>) {
+  return async function actionHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    const { action } = req.query;
+    const { data, slug } = req.body;
+
+    if (typeof action !== 'string') {
+      return res.status(400).json({ error: true, result: null });
+    }
+
+    let result;
+    try {
+      result = await fetcher.action({
+        actionId: action,
+        input: data,
+        slug,
+      });
+    } catch (error) {
+      return res.status(500);
```

Comment:
If we swallow the error here on the Next.js level, will it still log the error from cx-core somewhere for the developer to see? If not, I'd say we need to log the error here for troubleshooting purposes.